IAteIt/IAteIt/Application/IAteItApp.swift

```diff
@@ -14,15 +14,18 @@ struct IAteItApp: App {
     
     @StateObject var loginState: LoginStateModel = LoginStateModel()
     @StateObject var feedMeals: FeedMealModel = FeedMealModel()
+    @State private var isActive: Bool = false
     
     var body: some Scene {
         WindowGroup {
             NavigationView {
-                FeedView()
+                FeedView(isActive: $isActive)
```

Comment:
isActive 역할이 어떤거죵??!

Reply:
https://stackoverflow.com/a/61926030/18651361
위 링크 답변 마지막부분의 깃허브 링크를 보고 한건데, 이게 navigation 타고 들어간 뷰에서 root뷰로 나올 때 깊숙한(?) 뷰의 isActive 바인딩이 타고 들어간 모든 뷰에 적용되도록 해서 바로 root뷰로 pop될 수 있도록 하는 역할을 하는 것 같습니다!
(아래는 링크 질문의 첫번째 답변 인용)
> Along with setting `isDetailLink` to `false` on `NavigationLink`, pass the `isActive` binding to each subsequent destination view. At last when you want to pop to the root view, set the value to `false` and it will automatically pop everything off:

Reply:
오호라,, 요게 toggle 되는 시점이 언제인가요?? 코드로는 안보여서요!

Reply:
제가 참고한 건 stackoverflow 질문의 두번째 답변(environment 이용)인데 위 댓글에 인용한건 첫번째 답변(타고 들어가는 뷰들에 isActive 값을 binding으로 계속 연결)이라 이 경우에 맞는 설명이 아닌 것 같네요ㅜㅜ

isActive bool 값 자체는 NavigationLink 타고 들어가고 나갈때 바뀌는 것 같아요. 코멘트하신 line의 경우 FeedView에서 MyProfileView로 넘어갈때 true가 될 것 같습니당..@_@
```
NavigationLink(
    destination: MyProfileView().environmentObject(loginState),
    isActive: $isActive,
    label: { ProfilePhotoButtonView(loginState: loginState) }
)
.isDetailLink(false)
```